codegen/compile.py

```diff
@@ -0,0 +1,13 @@
+import os
```

Comment:
seems forgot to add this file
amended now, file is also moved to local/ folder as it is only used locally